### DISCUSSION TITLE
碰上无法导入类型该怎么办(?

### DISCUSSION THREAD

**Discussion by Rabbit0w0** [element-plus/element-plus#6832] [Q&A]
![unirhyme_3_25_2022](https://user-images.githubusercontent.com/21006278/160128106-1835ef48-b015-4541-8f64-c5089fb2e01d.png)
总之就是很奇怪的问题

**Comment by sxzz** [comment]
请提供具体报错，并确保为2.1.x版本

**Comment by KimYangOfCat** [comment]
也许有可能是语法提示插件的问题，推荐使用 volar
